charter.md

```diff
@@ -1,134 +1,144 @@
 # Charter of Data Distribution Working Group (DDWG)
 
+This charter outlines the necessary information that the DDWG requires to operate effectively.
+
+## Updates
+| Date | Changes |
+| --- | --- |
+| November 2013 | Initial Charter creation |
+| October 2018 | Hard drives no longer distributed regularly, BAS now distribution institution | 
```

Comment:
This year (2018) doesn't seem correct...

Reply:
Hey Evan, Yes that was a few years late :) but that is the last time the charter was updated.